### PR TITLE
Google Cloud Library

### PR DESCRIPTION
**Pull request by catlemus**
Add google cloud library so can now upload pictures using form

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ImageFormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ * Takes an image submitted by the user and uploads it to Cloud Storage, and then displays it as
+ * HTML in the response.
+ */
+@WebServlet("/upload")
+@MultipartConfig
+public class ImageFormHandlerServlet extends HttpServlet {
+
+    private static final long serialVersionUID = 9069792321854845875L;
+
+    @Override
```
Comment (emilyzhang242) [inline portfolio/src/main/java/com/google/sps/servlets/ImageFormHandlerServlet.java:29]: Nit: please align the @override with the function

File: portfolio/src/main/java/com/google/sps/servlets/ListImagesServlet.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Shows all of the images uploaded to Cloud Storage. */
+@WebServlet("/list-images")
+public class ListImagesServlet extends HttpServlet {
+
+    private static final long serialVersionUID = -7655683463700031557L;
+
+    @Override
```
Comment (emilyzhang242) [inline portfolio/src/main/java/com/google/sps/servlets/ListImagesServlet.java:20]: Nit: please align

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -51,6 +51,21 @@ <h3>
           Click <a href="pictures.html">here</a> to check out a gallery of my pictures. See my hiking trips,
           dogs, and favorite places!
       </p>
+
+      <!--picture upload form-->
+        <form action="/upload" method="POST" enctype="multipart/form-data">
+            <p>Upload one of your own favorite pictures!</p>
+            <input class = "inputI" type="file" name="image">
```
Comment (emilyzhang242) [inline portfolio/src/main/webapp/index.html:58]: What does inputI and submitI stand for?

**Review by emilyzhang242 — CHANGES_REQUESTED** [review]
Looks great overall, just please watch out for spacing and naming variable names in a descriptive manner!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (catlemus) [reply portfolio/src/main/webapp/index.html:58]: This is the inputted image and submit button for the image. I changed the names the inputImage and submitImage.

**Pull request merged**